### PR TITLE
Add TabPanels to ContactDetails

### PR DESCRIPTION
**Pull request by OzzieOrca**

### CODE REVIEW

File: src/components/Contacts/ContactDetails/ContactDetails.tsx [hunk 1]
```diff
@@ -50,11 +52,11 @@ export const ContactDetails: React.FC<Props> = ({
 }: Props) => {
   const { t } = useTranslation();
 
-  const [selectedTabIndex, setSelectedTabIndex] = React.useState(0);
+  const [selectedTabIndex, setSelectedTabIndex] = React.useState('0');
```
Comment (OzzieOrca) [inline src/components/Contacts/ContactDetails/ContactDetails.tsx:55]: Seemed like some of the typings for the tab components were expecting strings so I changed everything to a string.

**Comment by LizzyRandall** [comment]
Looks good to me. Do you know if you guys will have more tabs in the future? If so, maybe an enum would help then? I just think of Siebel we get more and more dumped on an account with different entities (notes, attachments, events, donations, activities... it's like 15 or so entities!)

**Comment by reldredge71** [comment]
This looks good to me!  I agree that an enum might be nice to have, I don't think we'll be adding to this tab bar too much, but I think enums are more precise than an index value.  I'm fine either way.

@OzzieOrca Do you know what's up with the eslint warning in the change log?

**Review by reldredge71 — APPROVED** [review]

**Comment by OzzieOrca** [comment]
Sure we can do an enum :)

**Pull request merged**